### PR TITLE
Update ingest issue template Dec 2023

### PR DESCRIPTION
**Pull request by climbfuji**
## Description

Add `geos ensemble`. Replace `Cheyenne` with `Casper/Derecho`, add `Hercules`, add `AWS ParallelCluster`

## Issue(s) addressed

n/a

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/data-ingest.md [hunk 1]
```diff
@@ -36,14 +36,16 @@ Data store directory path or S3 bucket path (_with wildcards if needed_):
...
 
 ## Data store checklist
 
 Data copied into R2D2 on:
 
-- [ ] Cheyenne
+- [ ] AWS ParallelCluster R&D
+- [ ] Casper/Derecho
 - [ ] Discover
 - [ ] Orion
```
Comment (climbfuji) [inline .github/ISSUE_TEMPLATE/data-ingest.md:50]:
```suggestion
- [ ] Orion/Hercules
```

**Review by shlyaeva — APPROVED** [review]
an uneducated review: I don't know much about where/what we ingest, but looks reasonable to me.

**Comment by ericlingerfelt** [comment]
My two cents.  

```
## Ingest description

*(Instructions: Please provide values for the following entries.)*

Description: 

Data type (_For example: observation, forecast_):

Other metadata (_For example: provider, observation_type, model_): 

Timeline (_When does this issue need to be completed_):

## Date range

Start date: 

End date:

## Source host and data store

Source host: 

Source data store:

## Target data store type (select one) 

- [ ] archive
- [ ] experiments
- [ ] gfsensemble
- [ ] geosensemble
- [ ] other 
    - Please describe: 

## Data ingest processing checklist for the R2D2 Administrator

Data has been ingested into the target data store type on the following data hubs:

- [ ] AWS ParallelCluster R&D
- [ ] Casper/Derecho (NCAR-Wyoming Supercomputing Center)
- [ ] Discover (NASA Center for Climate Simulation)
- [ ] Orion/Hercules (Mississippi State University High Performance Computing)
- [ ] S3 (jcsda-noaa-aws-us-east-1)
- [ ] S3 (jcsda-usaf-aws-us-east-2)
- [ ] S4 (Space Science and Engineering Center)
```

File: .github/ISSUE_TEMPLATE/data-ingest.md [hunk 2]
```diff
@@ -25,30 +25,32 @@ Start date:
...
-## Source host and directory/bucket
+## Source host and data store
 
 Source host: 
 
-Data store directory path or S3 bucket path (_with wildcards if needed_):
+Source data store:
 
-## Target data store type (select one)
+## Target data store type (select one)
```
Comment (climbfuji) [inline .github/ISSUE_TEMPLATE/data-ingest.md:34]:
```suggestion
## Target data store type (select one)
```

**Review by ericlingerfelt — APPROVED** [review]
Thanks, @climbfuji!

**Review by climbfuji — COMMENTED** [review]
self-review

**Pull request merged**